### ISSUE TITLE
D1. Definition of gcis:Publication

### ISSUE THREAD

**Issue by zednis** [USGCRP/gcis-ontology#32]
Definition of gcis:Publication is “A bounded physical representation of a body of information designed with the capacity (and usually intent) to communicate, and which has been made available to the public.”  Does a publication necessarily have to be bounded and physical?

**Comment by xgmachina** [comment]
This definition was copied from `bibo:Document`. We can consider an update to cover online and digital publicaitons.

```
<!-- http://purl.org/ontology/bibo/Document -->
<owl:Class rdf:about="Document">
<rdfs:comment xml:lang="en">A document (noun) is a bounded physical representation of body of information designed with the capacity (and usually intent) to communicate. A document may manifest symbolic, diagrammatic or sensory-representational information.
</rdfs:comment>
</owl:Class>
```

**Comment by justgo129** [comment]
I'm curious as to what "bounded" really means.  Within which constraints is it bounded?  Or, does bibo mean physically "bound?"  I ask because of the widespread use of electronic resources.

**Comment by xgmachina** [comment]
Perhaps it means 'complete', or 'versioned'?

**Comment by zednis** [comment]
I am not a fan of the definition from bibo.

The bibo ontology also states that `foaf:Document` is an equivalent class to `bibo:Document`

foaf document has the following definition with a note:

**definition:** _The Document class represents those things which are, broadly conceived, 'documents'._

**note:** _We do not (currently) distinguish precisely between physical and electronic documents, or between copies of a work and the abstraction those copies embody. The relationship between documents and their byte-stream representation needs clarification (see sha1 for related issues)._

...so electronic documents are valid `foaf:Document` instances, which means they are also valid `bibo:Document` instances.

**Comment by zednis** [comment]
I think we should change the definition for `gcis:Publication` so our intent is clear.

**Comment by zednis** [comment]
Also, for background knowledge we may want to think about how our class of Publication and its subclasses fit into the 4 levels of entities defined in the Functional Requirements for Bibliographic Records (FRBR)

https://en.wikipedia.org/wiki/Functional_Requirements_for_Bibliographic_Records
www.oclc.org/research/projects/frbr/

FRBR conceptualizes a spectrum of entities related to products of intellectual or artistic endeavor (e.g., publications).
- the **work**, a distinct intellectual or artistic creation
- the **expression**, the intellectual or artistic realization of a work
- the **manifestation**, the physical embodiment of an expression of a work
- the **item**, a single exemplar of a manifestation.

from the wikipedia link:

> Work is a "distinct intellectual or artistic creation."[1] For example, Beethoven's Ninth Symphony apart from all ways of expressing it is a work. When we say, "Beethoven's Ninth is magnificent!" we generally are referring to the work.
> 
> Expression is "the specific intellectual or artistic form that a work takes each time it is 'realized.'"[1] An expression of Beethoven's Ninth might be each draft of the musical score he writes down (not the paper itself, but the music thereby expressed).
> 
> Manifestation is "the physical embodiment of an expression of a work. As an entity, manifestation represents all the physical objects that bear the same characteristics, in respect to both intellectual content and physical form."[1] The performance the London Philharmonic made of the Ninth in 1996 is a manifestation. It was a physical embodiment even if not recorded, though of course manifestations are most frequently of interest when they are expressed in a persistent form such as a recording or printing. When we say, "The recording of the London Philharmonic's 1996 performance captured the essence of the Ninth," we are generally referring to a manifestation.
> 
> Item is "a single exemplar of a manifestation. The entity defined as item is a concrete entity."[1] Each copy of the 1996 pressings of that 1996 recording is an item. When we say, "Both copies of the London Philharmonic's 1996 performance of the Ninth are checked out of my local library," we are generally referring to items.

I am thinking the class `gcis:Publication` and its derivatives are aimed at an FRBR expression-level description - and I think we should attempt to make that clear in the definition.

**Comment by rewolfe** [comment]
In this sense, I believe we are using the term document to refer to its
Manifestation.  Even bits on a computer disk (or tape or in memory) are a
physical embodiment.

I believe we use the term bounded to refer to something that has a fixed
length.  It doesn't need to be something that has a binding (like a book).

On Thu, Jul 2, 2015 at 12:48 PM, Stephan Zednik notifications@github.com
wrote:

> Also, for background knowledge we may want to think about how our class of
> Publication and its subclasses fit into the 4 levels of entities defined in
> the Functional Requirements for Bibliographic Records (FRBR)
> 
> https://en.wikipedia.org/wiki/Functional_Requirements_for_Bibliographic_Records
> www.oclc.org/research/projects/frbr/
> 
> FRBR conceptualizes a spectrum of entities related to products of
> intellectual or artistic endeavor (e.g., publications).
> - the _work_, a distinct intellectual or artistic creation
> - the _expression_, the intellectual or artistic realization of a work
> - the _manifestation_, the physical embodiment of an expression of a
>   work
> - the _item_, a single exemplar of a manifestation.
> 
> from the wikipedia link:
> 
> Work is a "distinct intellectual or artistic creation."[1] For example,
> Beethoven's Ninth Symphony apart from all ways of expressing it is a work.
> When we say, "Beethoven's Ninth is magnificent!" we generally are referring
> to the work.
> 
> Expression is "the specific intellectual or artistic form that a work
> takes each time it is 'realized.'"[1] An expression of Beethoven's Ninth
> might be each draft of the musical score he writes down (not the paper
> itself, but the music thereby expressed).
> 
> Manifestation is "the physical embodiment of an expression of a work. As
> an entity, manifestation represents all the physical objects that bear the
> same characteristics, in respect to both intellectual content and physical
> form."[1] The performance the London Philharmonic made of the Ninth in 1996
> is a manifestation. It was a physical embodiment even if not recorded,
> though of course manifestations are most frequently of interest when they
> are expressed in a persistent form such as a recording or printing. When we
> say, "The recording of the London Philharmonic's 1996 performance captured
> the essence of the Ninth," we are generally referring to a manifestation.
> 
> Item is "a single exemplar of a manifestation. The entity defined as item
> is a concrete entity."[1] Each copy of the 1996 pressings of that 1996
> recording is an item. When we say, "Both copies of the London
> Philharmonic's 1996 performance of the Ninth are checked out of my local
> library," we are generally referring to items.
> 
> I am thinking the class gcis:Publication and its derivatives are aimed at
> an FRBR expression-level description - and I think we should attempt to
> make that clear in the definition.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/USGCRP/gcis-ontology/issues/32#issuecomment-118090507
> .

## 

Robert Wolfe, NASA GSFC @ USGCRP, o: 202-419-3470, m: 301-257-6966

**Comment by zednis** [comment]
In FRBR the bits on a particular disk would be the item and the PDF encoding (as a general concept) would be the manifestation.  I think in FRBR "physical embodiment" corresponds to 'format'.  The PDF and Word Document of a publication would be different manifestations in FRBR.  The PDF file on your local filesystem would be an item.

I think the best example is that a download link to a PDF file would represent a manifestation and the file you download to your local computer would be an item.

My guess is that our definition of publication aligns with FRBR work or expression - which I believe are closely aligned in the research publication domain because (I think) there is usually only 1 realization of the publication as an expression.  The separation between work and expression is greater for more general bibliographic needs because of how stories/works can be re-expressed with updated editions or in different mediums.

So back to our definition of Publication - I think it is important that a publication
- is a distinct work (perhaps this is a different way to say "bounded"?)
- with intent to communicate information
- that has been made available to the public

**Comment by rewolfe** [comment]
+1 - yes, that definition works for me.

On Mon, Jul 6, 2015 at 11:46 AM, Stephan Zednik notifications@github.com
wrote:

> In FRBR the bits on a particular disk would be the item and the PDF
> encoding (as a general concept) would be the manifestation. I think in FRBR
> "physical embodiment" corresponds to 'format'. The PDF and Word Document of
> a publication would be different manifestations in FRBR. The PDF file on
> your local filesystem would be an item.
> 
> I think the best example is that a download link to a PDF file would
> represent a manifestation and the file you download to your local computer
> would be an item.
> 
> My guess is that our definition of publication aligns with FRBR work or
> expression - which I believe are closely aligned in the research
> publication domain because (I think) there is usually only 1 realization of
> the publication as an expression. The separation between work and
> expression is greater for more general bibliographic needs because of how
> stories/works can be re-expressed with updated editions or in different
> mediums.
> 
> So back to our definition of Publication - I think it is important that a
> publication
> - is a distinct work (perhaps this is a different way to say
>   "bounded"?)
> - with intent to communicate information
> - that has been made available to the public
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/USGCRP/gcis-ontology/issues/32#issuecomment-118903282
> .

## 

Robert Wolfe, NASA GSFC @ USGCRP, o: 202-419-3470, m: 301-257-6966

**Comment by justgo129** [comment]
I'm fine with this, but am wondering whether we should clarify the meaning of "made available to the public?"  Many journal articles for instance are made available to the public but are not directly accessible without purchase or from most IP addresses.

**Comment by zednis** [comment]
How about just "made available" or "officially/formally released"?

**Comment by justgo129** [comment]
+1 for "made available."

**Comment by zednis** [comment]
definition: "a distinct work with intent to communicate information that has been made available"

It could use some tweaking for elegance but I think all the components are there.

**Comment by justgo129** [comment]
@zednis +1

**Comment by justgo129** [comment]
As I'm seeing no other opinions, @zednis please prepare the pull request.

**Comment by justgo129** [comment]
Closed #32 due to https://github.com/USGCRP/gcis-ontology/pull/66.
